easybuild/tools/multidiff.py

```diff
@@ -169,10 +169,13 @@ def get_line(self, line_no):
                     # track meta info (which filenames are relevant)
                     changes_dict.setdefault(diff_line, set()).add(meta)
 
-            # sort: lines with most changes last, limit number to MAX_DIFF_GROUPS
-            lines = sorted(lines, key=lambda line: len(changes_dict[line]))[:MAX_DIFF_GROUPS]
+            # sort lines with most changes last;
+            # sort lines with equal number of changes alphabetically to ensure consistent output;
+            # limit number to MAX_DIFF_GROUPS
+            lines = sorted(lines, key=lambda line: (len(changes_dict[line]), line))[:MAX_DIFF_GROUPS]
```

Comment:
I don't understand this? Your lambda returns a tuple? What does happen then by default if you compare tuples?